keras_cv/layers/object_detection/roi_generator_test.py

```diff
@@ -12,11 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import pytest
 import tensorflow as tf
 
 from keras_cv.layers.object_detection.roi_generator import ROIGenerator
 
 
+@pytest.mark.tf_keras_only
```

Comment:
For stuff like this I was also adding a runtime check, so users know what to do.

https://github.com/keras-team/keras-nlp/blob/c1eef6fe272cdf637b8833134f74b1fd815ea493/keras_nlp/utils/tensor_utils.py#L81-L86

Might be worth doing that in CV as well?

Reply:
Yeah I've added this with the `supports_ragged` flag for most affected cases. But I agree we should probably add a similar check to yet-unported components. I'll do that before I send this for review.

Reply:
oh whoops! didn't notice you hadn't sent yet, was just scanning the top of PRs :)

Reply:
All good 😄